index.html

```diff
@@ -0,0 +1,113 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com"> 
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin> 
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;500;700&display=swap" rel="stylesheet">
+    <link
+      href="//db.onlinewebfonts.com/c/23c0fcab84d99da0de762de7e220a6e1?family=Europa"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link
+      href="http://fonts.cdnfonts.com/css/eurostile-extended"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+  </head>
+  <body>
+    <nav>
+      <img src="./images/mf-logo.svg" alt="" class="brand-img" />
+
+      <div class="nav-right">
+        <span class="nav-text">Mentorship</span>
+        <button class="nav-button">Say Hello</button>
+      </div>
+    </nav>
+
+    <main>
+      <h1>Front-end Developer</h1>
+      <p class="main-p">
+        I design and code beautifully simple things, and I love what I do
+      </p>
+      <img src="./images/mf-avatar.svg" alt="Graphic-profile-pic" class="img-avatar" />
+      <br />
+      <img src="./images/hero.svg" alt="Coding process" class="img-machine" />
+    </main>
+    <section class="personal-info-container">
+      <div class="text">
+        <h2>Hi, I'm Rami. Nice to meet you</h2>
+        <p>
+          From when I was a teenager I was interested in Web development, especially 
+          the front side of it since I prefer visuals and instantly seeing the changes
+          that I am doing. I'm currently doing an internship that is teaching me new things
+          and allowing me to sharpen my skills. After it, I would like to graduate from Computer
+          Science and start my career.
+        </p>
+      </div>
+    </section>
+
+    <div class="card-info">
+      <div class="card-info-skills">
+          <img src="./images/skills.svg" alt="">
+          <h2>Front-end Developer</h2>
+          <p class = "p-personal">I like to code things from scratch, and enjoy bringing ideas to life in the browser.</p>
+          <h3>Languages I speak:</h3>
+          <p class = "p-skills">HTML, CSS, JavaScript</p>
+          <h4>Dev Tools:</h4>
+          <ul>
+            <li>Git</li>
+            <li>GitHub</li>
+            <li>Bootstrap</li>
+          </ul>
+      </div>
+    </div>
+        
+    <section class = "recent-work-container">
+      <h2 class="recent-work-container__h2">My Recent Work</h2>
+      <p class="recent-work-container__p">Here are a few design projects that I have worked on</p>
+      <div class="project-grid">
+          <div class="project-grid__item">
```

Comment:
mentioned before, concatination of divs

Reply:
Also here I need the divs in order to have each card have grid applied to it. And the project-overlay div is for the hover animation.

I don't know if there is another way to do it other than using divs. If there is I would love to know :D

Reply:
grid is a bit outdated consider usign flex as mentioned in another comment